tools/airgap_linter.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python3
+#
+# Validates that the supplied framework (identified by the directory passed in) will function, to the best
+# of our knowledge, in an airgapped cluster.
+#
+# The checks are:
+#   - No external URIs defined in anything but resource.json
+#   - No images defined in anything but resource.json (and templated into the svc.yml)
+#
+
+import re
+import sys
+import os
+
+
+def extract_uris(file_name):
+    with open(file_name, "r") as file:
+        lines = file.readlines()
+
+    matcher = re.compile(".*(https:\/\/|http:\/\/)([^\/]*)", re.IGNORECASE)
```

Comment:
I'm hesitant to change a regex that works (or at least mostly works. I'm making a minor edit to it. Let me know what you think (patch coming soon)